osgp/platform/osgp-adapter-domain-core/src/main/java/org/opensmartgridplatform/adapter/domain/core/infra/jms/core/messageprocessors/CommonRegisterDeviceRequestMessageProcessor.java

```diff
@@ -7,42 +7,26 @@
  */
 package org.opensmartgridplatform.adapter.domain.core.infra.jms.core.messageprocessors;
 
-import javax.jms.JMSException;
 import javax.jms.ObjectMessage;
 
-import org.opensmartgridplatform.adapter.domain.core.application.services.FirmwareManagementService;
 import org.opensmartgridplatform.adapter.domain.core.infra.jms.DomainCoreDeviceRequestMessageProcessor;
-import org.opensmartgridplatform.shared.infra.jms.MessageMetadata;
 import org.opensmartgridplatform.shared.infra.jms.MessageType;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
 public class CommonRegisterDeviceRequestMessageProcessor extends DomainCoreDeviceRequestMessageProcessor {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CommonRegisterDeviceRequestMessageProcessor.class);
 
-    @Autowired
-    private FirmwareManagementService firmwareManagementService;
-
     public CommonRegisterDeviceRequestMessageProcessor() {
         super(MessageType.REGISTER_DEVICE);
     }
 
     @Override
-    public void processMessage(final ObjectMessage message) throws JMSException {
-
-        MessageMetadata messageMetadata;
-        try {
-            messageMetadata = MessageMetadata.fromMessage(message);
-        } catch (final JMSException e) {
-            LOGGER.error("UNRECOVERABLE ERROR, unable to read ObjectMessage instance, giving up.", e);
-            return;
-        }
-
-        this.firmwareManagementService.handleSsldPendingFirmwareUpdate(messageMetadata.getDeviceIdentification());
+    public void processMessage(final ObjectMessage message) {
+        LOGGER.info("REGISTER_DEVICE message received. Ignore, since we act on CONFIRM_REGISTER_DEVICE");
```

Comment:
If we choose to keep the message sent from osgp-core to domain-core, this logging can be set to debug in my opinion. Or we could remove this message processor altogether, and remove the sending of the message in osgp-core.

Reply:
In the latter case: how will we handle    this.firmwareManagementService.handleSsldPendingFirmwareUpdate(messageMetadata.getDeviceIdentification());
in CommonConfirmRegisterDeviceRequestMessageProcessor? Where will we handle pending firmware updates?

Reply:
The pending firmware records are checked by the implementation of the other message processor.